### ISSUE TITLE
Migrations context-menu is missing

### ISSUE THREAD

**Issue by gilyadid** [ErikEJ/EFCorePowerTools#2434]
Migrations context-menu is missing after upgrading to latest version 2.6.398.
The migration context-menu existed before the upgrade.
There is no option to rollback since it is a vs extension and not a project extension.

I tried to uninstall and reinstall the extension but that did not resolve it.

Technical details:
- EF Core Power Tools version: 2.6.398
- Exact Visual Studio version: Microsoft Visual Studio Enterprise 2022 (64-bit) - Version 17.10.4
- Database engine: SQL Server
- EF Core version in use: EF Core 8.0.7
- Is Handlebars templates used: no
- Is T4 templates used: no
- Is a SQL Server .dacpac used: no

**Comment by ErikEJ** [comment]
Yes, that feature has been removed, something similar is now built into VS.

**Comment by gilyadid** [comment]
Thanks Erik, found it. 
For others who maybe looking for it: https://www.youtube.com/watch?v=j2XuiWq9Qes
